src/CADL.Extension/readme.md

```diff
@@ -0,0 +1,143 @@
+# CADL to .Net
+
+DPG 1.1 requires CADL as input, if service would like to generate models.
+
+## Pipeline
+
+Cadl->net.emitter-> json -> outputLibrary -> sdk
+
+![E2E flow](flow.png)
+
+## Components
+
+### csharp emitter
+
+The emitter responses to emit the cadl model as json from cadl file.
```

Comment:
Don't quite understand this line.  Do you mean,

'The emitter will emit the cadl model into json file from input CADL file.'  ?